### ISSUE TITLE
mdx files get // comments converted to /

### ISSUE THREAD

**Issue by lukeapage** [trivago/prettier-plugin-sort-imports#114]
**Your Environment**
 * **Prettier version**: 2.4.1
 * **node version** [12.x.x, 14.x.x]: 16.13.0
 * **package manager**: yarn@1
 * **IDE**: Webstorm

**Describe the bug**

A `mdx` file:

```
import Heading from './heading';

export const Text = ({ className, children }) => (
    // use <section> instead of <div> so storybook doesn't style it automatically
    <section className={className}>{children}</section>
);

<Meta title='Heading' component={Heading} />

<Story
    name='Heading'
    parameters={{
        docs: { disable: true },
    }}
    args={{
        type: 'primary',
        size: 'medium',
        isCenter: false,
        children: 'Heading',
    }}
>
    {(args) => <Heading {...args} />}
</Story>

# Heading

Headings...
```

**To Reproduce**

```
git clone https://github.com/lukeapage/mdx-prettier-bug.git
cd mdx-prettier-bug
yarn
yarn prettier --write test.mdx
git diff
```

**Expected behavior**

For the slashes in comments not to be removed.

If I remove this plugin, no slashes are removed.

**Contribute to @trivago/prettier-plugin-sort-imports**

- [ ] I'm willing to fix this bug 🥇

**Comment by lukeapage** [comment]
The output code from this plugin looks correct.. which means that it is prettier that is broken.

**Comment by lukeapage** [comment]
Raised a bug for prettier after trying to work it out: https://github.com/prettier/prettier/issues/11859

I still don't think its the plugin, but I also am confused as to why adding the plugin makes it break, considering where I tracked it down to.

**Comment by ayusharma** [comment]
We are aware of this issue. We faced the exact same issue and tried to solve it in prettier. Thank you so much for reporting it. 😃

**Comment by lukeapage** [comment]
This appears fixed in 2.5.0! :)
